opencti-platform/opencti-graphql/src/config/errors.js

```diff
@@ -1,43 +1,47 @@
-import { createError } from 'apollo-errors';
+import { ApolloError, createError } from 'apollo-errors';
 
-const CATEGORY_TECHNICAL = 'technical';
-const CATEGORY_BUSINESS = 'business';
+const CATEGORY_TECHNICAL = 'TECHNICAL';
```

Comment:
Question: why don't we use the exception type to set the error category instead of having constant here ? We could also set the category inside the error type.